config/locales/app/views/activerecord.en.yml

```diff
@@ -0,0 +1,10 @@
+en:
+  activerecord:
+    errors:
+      messages:
+        too_short: "is too short (minimum is %{count} characters)"
+        no_special_char: "must include at least one special character"
+        no_digit: "must include at least one digit"
+        no_uppercase: "must include at least one uppercase letter"
+        no_lowercase: "must include at least one lowercase letter"
+        terms_of_use_acceptance_required: "You cannot register without accepting the terms of use."
```

Comment:
Bu validation mesajı globalde belirtmeye gerek yok. Bunu user model'i için olan translation içinde yapabiliriz. Orada terms_of_use altınca acceptance hata mesajı ile verebiliriz. Global'e çıkartmayalım.